src/main/java/com/skeleton/user/service/UserService.java

```diff
@@ -59,4 +73,12 @@ private String getRandomCode() {
         }
         return sb.toString();
     }
+
+    private void verifyUserPassword(UserVerifyRequest request, User user) {
```

Comment:
```suggestion
    private void verifyUserPassword(UserVerifyRequest request, String password) {
```

`verifyUserPassword` 메서드 내에서 user는 비밀번호 조회 시에만 이용되어서 user 객체가 아니라 **패스워드만 넘기는** 편이 결합도를 낮출 수 있을 것 같습니다! 또 엔티티를 직접 넘기지 않아서 혹시라도 비밀번호가 수정되거나 하는 일도 방지할 수 있을 것 같아요.

Reply:
또 지금 /verify 에서 성공시 `200(ok)` 를 리턴하는데 response가 없으니 `204(no content)` 가 더 명확하지 않을까 생각합니다

Reply:
마지막으로 메서드를 `PUT`으로 하신 이유가 궁금합니다. PUT을 쓰면 리소스를 새로 만들거나 전체 값을 대체할 때 사용한다고 알고 있어서요. userVerification 필드만 수정하는거니 PATCH나 POST가 맞지 않나 생각했습니다.

Reply:
코멘트 감사합니다!! 우선 다른 인증번호를 입력해도 승인이 되는 문제는 Redis같은 저장소에 저장하지 않아서, DB에 저장하기 보다는 일단 응답으로만 내리자는 생각에 그렇게 구현했습니다. 생각해보니 이건 DB에 인증번호를 저장하고 인증완료되면 null처리를 하던가 하는 방식으로라도 구현하는게 좋겠다고 생각하는데 어떤가요??

Reply:
두번째 코멘트에서도 password만 넘기는게 좋겠어요.리턴도 204사용하고, 메서드 PATCH매핑이 더 적절하겠네요!!

Reply:
> 코멘트 감사합니다!! 우선 다른 인증번호를 입력해도 승인이 되는 문제는 Redis같은 저장소에 저장하지 않아서, DB에 저장하기 보다는 일단 응답으로만 내리자는 생각에 그렇게 구현했습니다. 생각해보니 이건 DB에 인증번호를 저장하고 인증완료되면 null처리를 하던가 하는 방식으로라도 구현하는게 좋겠다고 생각하는데 어떤가요??

아 나중에 redis를 쓴다면 굳이 저장 안하는게 맞을것 같긴하네요!  메인 DB에 인증번호 테이블 따로 할당하는건 좀 공간 낭비 DB 트래픽 낭비 같고, 또 인증번호는 redis같은 인메모리 DB를 써서 3분동안만 유효하고 그 다음에는 삭제되는... 그런 방식으로 시스템을 구현하는 게 좋을 것 같습니다.!😄

Reply:
그래서 저장을 안하면, 다른 인증코드가 입력되어도 가입이 되는데... 일단 둘까요?

Reply:
음.. 저는 일단 둬도 된다고 생각합니다!  다른 팀에서 redis를 써서 인증코드 api를 만들고, 그쪽에서 sms발송이라던가 인증코드 확인을 한다고 생각하면... 저희는 그쪽에서 인증이 성공했는지 아닌지만 받아서 DB에 저장하면 같네요. 멘토님이 혹시 질문하신다고 해도 대충 이럴 예정이다하면 넘어가실것 같구요ㅎㅎ...

Reply:
답변 너무 감사드립니다~ 그럼 그대로 두겠습니다!!